scripts/server.py

```diff
@@ -0,0 +1,115 @@
+"""
+Before running server:
+    set up and run postgres server with database `artifact` and table `test_name`
+    run `echo DATABASE_URL=postgres://username:password@localhost/artifact > .env'
+        to create .env file in /artifact directory
+    (the database name might be changeable, as long as it is specified in the .env)
+
+Run the artifact server with:
+    cargo run --features server -- -v serve
+
+When running this script in interactive mode you can access all functions through:
+    create, read, update and delete
+or directly through the object:
+    api
+"""
+
+from __future__ import print_function
+
+import argparse
+import code
+import pprint
+
+from JsonRpc import JsonRpc
+
+
+class CrudApi(JsonRpc):
+    def create_art(self, params):
+        payload = {
+            'method': 'CreateArtifacts',
+            'params': params,
+        }
+        self.post(self.host, json=payload)
+
+    def read_art(self, params):
```

Comment:
As of now, this implementation is not very user-friendly anyways.
The functions should allow more introspection into the actual API parameters than just blindly submitting `params`. Since I don't know how `params` are actually structured for each respective RPC I'd just leave it in like this or make `params` optional.